src/services/shims.ts

```diff
@@ -693,10 +693,10 @@ namespace ts {
             return realizeDiagnostics(diagnostics, newLine);
         }
 
-        public getSyntacticClassifications(fileName: string, start: number, length: number): string {
+        public getSyntacticClassifications(fileName: string, start: number, length: number, format?: SemanticClassificationFormat): string {

```

Comment:
why is SemanticClassificationFormat sent to SyntacticClassifications ?

Reply:
Great point Sheetal, Thanks! I was fidgeting around and left this in by accident - the only changes needed are now in the PR